src/main/java/com/example/api/controllers/MessageController.java

```diff
@@ -34,6 +36,19 @@ public ResponseEntity<Object> show(@PathVariable("messageId") int messageId)
         }
     }  
 
+    @PutMapping("/{messageId}")
+    public ResponseEntity<Object> update(@PathVariable("messageId") int messageId, @RequestBody Message updatedMessage) {
```

Comment:
```suggestion
    public ResponseEntity<Object> update(@PathVariable("messageId") int messageId, @RequestBody Message updatedMessage) {
```

Utiliser un DTO.
Lors de l'update, si il n'y a rien dans la requete, alors le message doit quand meme etre renvoyé sans modification